presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -108,6 +108,12 @@ public static PlanMatchPattern tableScan(String expectedTableName)
         return node(TableScanNode.class).with(new TableScanMatcher(expectedTableName));
     }
 
+    public static PlanMatchPattern tableScan(String expectedTableName, String originalConstraint)
+    {
+        Expression expectedOriginalConstraint = rewriteIdentifiersToSymbolReferences(new SqlParser().createExpression(originalConstraint));
```

Comment:
he `originalConstraint` field bears the following comment, are you sure you want to test it?

> HACK! This field exists for the sole purpose of being able to print the original predicates (from the query) in a human readable way.